resources/js/src/app/components/itemList/filter/ItemCategoryDropdown.spec.ts

```diff
@@ -0,0 +1,206 @@
+        const facet: Facet = {
+            cssClass: '',
+            findologicFilterType: 'select',
+            id: 'cat',
+            isMain: false,
+            itemCount: 6,
+            name: 'Category',
+            noAvailableFiltersText: '',
+            select: 'single',
+            type: '',
+            values: [
+                {
+                    count: 4,
+                    id: '4',
+                    image: '',
+                    items: [
+                        {
+                            count: 2,
+                            id: '5',
+                            image: '',
+                            items: [],
+                            name: '"Armchairs & Stools"',
+                            position: 'item',
+                            selected: false
+                        },
+                        {
+                            count: 2,
+                            id: '6',
+                            image: '',
+                            items: [],
+                            name: 'Sofas',
+                            position: 'item',
+                            selected: false
+                        }
+                    ],
+                    name: 'Living Room',
+                    position: 'item',
+                    selected: false
+                },
+                {
+                    count: 4,
+                    id: '5',
+                    image: '',
+                    items: [
+                        {
+                            count: 2,
+                            id: '8',
+                            image: '',
+                            items: [],
+                            name: 'Something',
+                            position: 'item',
+                            selected: false
+                        },
+                        {
+                            count: 2,
+                            id: '9',
+                            image: '',
+                            items: [],
+                            name: 'Something else',
+                            position: 'item',
+                            selected: false
+                        }
+                    ],
+                    name: 'Living Room',
+                    position: 'item',
+                    selected: false
+                }
+            ]
+        };
+
+        store.state.itemList = {
+            isLoading: false,
+            selectedFacets: [],
+            facets: [facet]
+        };
+
+        const wrapper = shallowMount(ItemCategoryDropdown, { propsData: { facet }, store, localVue });
+
+        expect(wrapper.element.querySelectorAll(':scope > div.fl-category-dropdown-container.custom-select > ul > li').length).toBe(2);
+        expect(wrapper.element.querySelectorAll(':scope > div.fl-category-dropdown-container.custom-select > ul > li ul.subcategories li').length).toBe(0);
```

Comment:
Probably preference but I prefer this as it is a bit better readable:
```suggestion
        expect(wrapper.find(':scope > div.fl-category-dropdown-container.custom-select > ul > li ul.subcategories li').exists()).toBeFalsy();
```